### PR TITLE
README: point to WG draft, not draft-wussler

### PR DESCRIPTION
**Pull request by dkg**

### CODE REVIEW

**Comment by dkg** [comment]
probably worth having some more cleanup on the editor's copy stuff as well.

**Review by falko-strenzke — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 This is the working area for the individual Internet-Draft, "Post-Quantum Cryptography in OpenPGP".
 
 * [Editor's Copy](https://openpgp-pqc.github.io/draft-openpgp-pqc/#go.draft-wussler-pqc-openpgp.html)
```
Comment (fluppe2) [inline README.md:5]:
```suggestion
* [Editor's Copy](https://openpgp-pqc.github.io/draft-openpgp-pqc/#go.draft-ietf-openpgp-pqc.html)
```

File: README.md [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
-* [Datatracker Page](https://datatracker.ietf.org/doc/draft-wussler-openpgp-pqc)
-* [Individual Draft](https://datatracker.ietf.org/doc/html/draft-wussler-openpgp-pqc)
+* [Datatracker Page](https://datatracker.ietf.org/doc/draft-ietf-openpgp-pqc)
+* [Individual Draft](https://datatracker.ietf.org/doc/html/draft-ietf-openpgp-pqc)
 * [Compare Editor's Copy to Individual Draft](https://openpgp-pqc.github.io/draft-openpgp-pqc/#go.draft-wussler-openpgp-pqc.diff)
```
Comment (fluppe2) [inline README.md:8]:
```suggestion
* [Compare Editor's Copy to Individual Draft](https://openpgp-pqc.github.io/draft-openpgp-pqc/#go.draft-ietf-openpgp-pqc.diff)
```

**Comment by fluppe2** [comment]
> probably worth having some more cleanup on the editor's copy stuff as well, but i don't understand that setup well enough to offer a specific edit.

just did it (tried cleaned up links in the browser first)

**Review by fluppe2 — APPROVED** [review]
Applied further cleanups

**Pull request merged**